### DISCUSSION TITLE
Next task?

### DISCUSSION THREAD

**Discussion by antoniokunstek** [foivz/pop-app-backend#29] [Q&A]
Dear @mislavmatijevic  (TOP G 😎)

What assignment will you take next? What task do you want me to take on?
Perhaps it would be better to make an agreement earlier, just to have as few conflicts as possible with controllers and services.

**Comment by mislavmatijevic** [comment]
Dear @antoniokunstek (KinG 👑)

I suggest we avoid #11, #13, #16, #17 until their required funtionality is added by the preceeding issues.
Always feel free to pick from any existing issues. I would only advise against taking more than 1 issue at a time.

I'll take a new issue probably later today.
